tools/webdashboards/add_new_file.md

```diff
@@ -7,7 +7,7 @@ This task can be split into smaller tasks:
 * Specify the list of locales supported by this file (mandatory in most occasions).
 * Add the files to all locales in the l10n repository.
 
-To analyze all steps, let’s consider a practical example: we need to add a file called `mozorg/contribute/signup.lang` to the project `www.mozilla.org`. The file is `critical` only for French and German, `opt-in` for other locales, and deadline is 30 May 2016. Most of the time a file will be critical for all locales, or won’t have any flags.
+To analyze all steps, let’s consider a practical example: you need to add a file called `mozorg/contribute/signup.lang` to the project `www.mozilla.org`. The file is `critical` only for French and German, `opt-in` for other locales, and deadline is 30 May 2016. Most of the time a file will be critical for all locales, or won’t have any flags.
```

Comment:
Nitpicking: 30 May 2016 doesn't sound natural to me in that order. Probably better to change to May 30th or 30th of May 2016